pkg/planner/core/access_object.go

```diff
@@ -283,6 +283,19 @@ func (p *PhysicalTableScan) AccessObject() base.AccessObject {
 			res.Partitions = []string{partitionName}
 		}
 	}
+	if p.AnnIndexExtra != nil {
+		index := IndexAccess{
+			Name: p.AnnIndexExtra.IndexInfo.Name.O,
+		}
+		for _, idxCol := range p.AnnIndexExtra.IndexInfo.Columns {
+			if tblCol := p.Table.Columns[idxCol.Offset]; tblCol.Hidden {
+				index.Cols = append(index.Cols, tblCol.GeneratedExprString)
```

Comment:
Vector indexes haven't hidden columns, currently only expression indexes seem to be used.